### DISCUSSION TITLE
Possible to have multiple GitHub Vercel Apps on one account?

### DISCUSSION THREAD

**Discussion by ehowey** [vercel/community#562] [Help] [answered]
Hello!

I have a bit of an odd scenario and a question about it. I do freelance work, my latest client wanted to take over managing and billing for hosting/database, etc. I don't normally do that, but did it in this case, might not do it again.

So I had them create a Vercel Pro account, linked it to the repo which I have on my own GitHub account. I then restricted access for the Vercel App to only that select repo. I don't want them to have access to all of the different repos I have.

Now herein lies the problem, I want my own Vercel Pro account to have total access to all of my GitHub repos. I can't see a way to add a second Vercel app to my GitHub account. Any way to do this? Or any way around this?

**Comment by samcx** [comment, marked as answer]
@ehowey Unfortunately, there is no way around this. You will have to create another Vercel account for a different GitHub Login Connection / GitHub Integration.
